frontend/public/components/utils/resource-link.jsx

```diff
@@ -12,7 +12,7 @@ import { FLAGS } from '../../const';
 const unknownKinds = new Set();
 
 export const resourcePathFromModel = (model, name, namespace) => {
-  const {path, namespaced, crd} = model;
+  const {plural, namespaced, crd} = model;
```

Comment:
@spadgett Hi, this function is building url for the console ( vs. building for the k8s API ) , shouldn't is use `plural` instead of `path`, what am I missing ?

Reply:
Models representing custom resources (having `crd: true`) will have their `resourcePath` based on `referenceForModel` function (`[group]~[version]~[kind]`), which serves as the key for all CRD-based models.

Models representing core k8s objects (without `crd: true`) will have their `resourcePath` based on model's `path` attribute only.

Model's `plural` attribute seems to be used for organizing models within Redux and k8s queries, see e.g. `connectToPlural` in `public/kinds.ts`.